config/maintainers.json

```diff
@@ -22,10 +22,10 @@
     },
     {
       "github_username": "NobbZ",
-      "show_on_website": false,
+      "show_on_website": true,
       "alumnus": false,
-      "name": null,
-      "bio": null,
+      "name": "Norbert Melzer",
+      "bio": "Found out about erlang via exercism, got hooked up and after fixing some trivial bugs, I have been added to the team,",
```

Comment:
Then its probably “hooked”, rather than “hooked up”.

Also I'll think about some more detailed text, but how much space will there roughly on the “card” for the bio? How will linebreaks in the strings be handled? How well does JSON multi-line-strings in general?